ChangeLog

```diff
@@ -17,6 +17,7 @@
  * Removed deprecated HSICStat ctors relying on weight matrix
 
 === Documentation ===
+ * #2339 (FunctionalChaosSobolIndices has documentation issues)
```

Comment:
there is no bugfixes list anymore, thats because we agreed that bugfixes list must be completed at once on the rc, to avoid conflicts

Reply:
Fixed at a2d147e0d737c66d99f5c140da909bcda17abad8.